### PR TITLE
Puppeteer fix

### PR DESCRIPTION
**Pull request by nadvolod**
Removing broken Puppeteer code
Pointing to a better repo that isn't maintained by us

### CODE REVIEW

File: testrunner-toolkit/puppeteer/README.md [hunk 1]
```diff
@@ -1,54 +1,5 @@
...
-
-Test Suites: 8 passed, 8 total
-Tests:       25 passed, 25 total
-Snapshots:   0 total
-Time:        49.838s
-Ran all test suites.
-
-Open job details page: https://app.eu-central-1.saucelabs.com/tests/773b6c6d0e8d4bc7a4cba2def2b10903
-```
+* [Docs](https://docs.staging.saucelabs.net/testrunner-toolkit/index.html)
```
Comment (wswebcreation) [inline testrunner-toolkit/puppeteer/README.md:3]: @nadvolod 

People don't have access to staging 😉

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @nikolay-advolodkin 

thanks for the fix, can we just completely remove it? AFAIK we can't run it on Sauce, only with Docker, secondly it's not used a lot

File: testrunner-toolkit/puppeteer/README.md (hunk 1 above)
Comment (nadvolod) [reply testrunner-toolkit/puppeteer/README.md:3]: @wswebcreation I'll fix the staging but I don't think we should remove it.
A. There are people that come to `demo-js` for all things related to Sauce JS
B. I'm doing a demo with customers on Friday in regards to this exact tech stack

Comment (nadvolod) [reply testrunner-toolkit/puppeteer/README.md]: @wswebcreation PS, I don't use @nikolay-advolodkin account, Sauce created it

**Pull request merged**